build.gradle

```diff
@@ -1,9 +1,9 @@
 plugins {
     id 'java'
-    id 'maven'
+    id 'maven-publish'
```

Comment:
Reason for maven->maven-publish:
[Official Gradle Docs](https://docs.gradle.org/7.0/userguide/upgrading_version_6.html?_ga=2.58294250.1315200260.1696130240-1061710299.1696130240&_gl=1*1bvafky*_ga*MTA2MTcxMDI5OS4xNjk2MTMwMjQw*_ga_7W7NC6YNPT*MTY5NjEzMDI0MC4xLjEuMTY5NjEzMjY2Ni4zLjAuMA..#removal_of_the_legacy_maven_plugin)

Just for knowing why :)


[Original Forum Discussion](https://discuss.gradle.org/t/plugin-with-id-maven-not-found/39621)

Reply:
Please resolve this comment since I just wanted to point out to people that might be curious.